templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,13 @@ <h1>
     </a>
   </h1>
 
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="🔎" class="btn btn-secondary">
+  </form>
+
+  <br>
```

Comment:
Create additional include html for this. And just reuse it

Reply:
Done! Thank you :)